frontend/src/main/scala/bloop/Cli.scala

```diff
@@ -257,7 +257,6 @@ object Cli {
         .flatMap(start => taskState.materialize.map(s => (s, start)))
         .map { case (state, start) => logElapsed(start); state }
         .dematerialize
-        .executeWithOptions(_.enableAutoCancelableRunLoops)
```

Comment:
We don't need cancel to propagate through the current flatMap stack, so we disable this. I've also experienced some weird Monix behaviour when this is enabled. Since this is not a well-tested feature and it doesn't make much sense to have it enabled by default, I'm more comfortable with using the stock task options.